### PR TITLE
wirte Controller, Service and View for display a note

### PR DESCRIPTION
**Pull request by pushpalata06**
@kalehrishi 
@amitheda

### CODE REVIEW

**Comment by pushpalata06** [comment]
@amitheda

File: app/Controller/Web/Note.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        $this->view    = new View();
+    }
+    public function get()
+    {
+        $sessionModel = new SessionModel();
+        $sessionModel->setUserId($this->request->getCookies()['userId']);
+        $sessionModel->setAuthToken($this->request->getCookies()['authToken']);
+        
+        $sessionService = new SessionService();
+        try {
```
Comment (kalehrishi) [inline app/Controller/Web/Note.php]: everything under single try catch is not good idea.

File: app/Controller/Web/Note.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            } catch (ModelNotFoundException $error) {
+                $response    = $error->getMessage();
+                $this->view->render("Note.php", $response);
+            }
+            $notesArray = $noteModel->toArray();
+            $this->view->render("Note.php", $notesArray);
+            return $noteModel;
+        } catch (ModelNotFoundException $error) {
+            $app = \Slim\Slim::getInstance();
+            $app->redirect("/error");
```
Comment (kalehrishi) [inline app/Controller/Web/Note.php]: We should redirect user to login.

File: app/Controller/Web/Note.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            $noteModel->setId($noteId);
+
+            $noteService =  new NoteService();
+            try {
+                $noteModel = $noteService->get($noteModel);
+            } catch (ModelNotFoundException $error) {
+                $response    = $error->getMessage();
+                $this->view->render("Note.php", $response);
+            }
+            $notesArray = $noteModel->toArray();
```
Comment (kalehrishi) [inline app/Controller/Web/Note.php]: should pass model to view instead of array

**Pull request merged**